SeaPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md

```diff
@@ -147,7 +147,60 @@ A general description of SEA and MM supervisor code integration is depicted belo
 > Note: A list of the libraries and modules made available by this package is provided in the
   [Software Component Overview](SoftwareComponentOverview.md).
 
-### Platform DSC statements
+### Build Flow
+
+Building a platform that integrates the SEA core and and MM supervisor is a multi-stage process as the MmSupervisorCore
+must be built before the Stm (and optionally the rest of the platform). This is due to the fact that the Stm consumes

```

Comment:
I suggest capitalizing the acronym throughout to make it clear it is any acronym.

```suggestion
must be built before the STM (and optionally the rest of the platform). This is due to the fact that the STM consumes
```

Reply:
Sounds good. I updated stm or Stm to `STM` throughout the document (ignoring any usage of Stm.bin or Stm.inf)